arroyo/backends/kafka/consumer.py

```diff
@@ -628,6 +628,11 @@ def close(self, timeout: Optional[float] = None) -> None:
     def closed(self) -> bool:
         return self.__state is KafkaConsumerState.CLOSED
 
+    @property
+    def member_id(self) -> str:
```

Comment:
Maybe we should also add `@cached_property`, depending on the cost?

```suggestion
    @cached_property
    def member_id(self) -> str:
```

Reply:
I think it cheap. Also, I believe the member id changes depending on the rebalancing strategy used. So caching might be inacurate.